src/modules/cli/lib/install-macos.ts

```diff
@@ -1,19 +1,26 @@
 import { dialog } from 'electron';
-import { mkdir, readlink, symlink, unlink } from 'node:fs/promises';
+import { mkdir, readlink, symlink, unlink, lstat } from 'node:fs/promises';
 import path from 'node:path';
 import * as Sentry from '@sentry/electron/main';
-import { __ } from '@wordpress/i18n';
+import { __, sprintf } from '@wordpress/i18n';
 import { sudoExec } from 'src/lib/sudo-exec';
 import { getMainWindow } from 'src/main-window';
 import { getResourcesPath } from 'src/storage/paths';
 import packageJson from '../../../../package.json';
 
-const cliSymlinkSource = '/usr/local/bin/studio';
+const cliSymlinkPath = '/usr/local/bin/studio';
 
 const binPath = path.join( getResourcesPath(), 'bin' );
-const cliSymlinkTarget = path.join( binPath, 'studio-cli.sh' );
+const cliPackagedPath = path.join( binPath, 'studio-cli.sh' );
 const installScriptPath = path.join( binPath, 'install-studio-cli.sh' );
 
+enum InstallError {
```

Comment:
Hm, it seems, I remember that either Tumblr or Dotcom or both didn't recommend using enums, due to tree shaking issues or something like that. Do you remember something about it? Or maybe I remember something incorrectly 😅

Reply:
Let's wrap strings with translation function.

Reply:
These error messages aren't displayed to users, so we shouldn't need to translate them. You're right that we don't need an enum, though. I've changed it to regular constants.